From 7f5869cd32a7b3f717d9544b35562d9d01ca6510 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Wed, 5 Jul 2017 03:03:16 -0400
Subject: [PATCH 14/14] Always use devmapper for kpartx

Do not try to compute the LIBDM_API_COOKIE make variable
from host information when cross-compiling.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Aws Ismail <aws.ismail@windriver.com>
Signed-off-by: Joe Slater <joe.slater@windriver.com>

Rebase to 0.7.1
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 kpartx/Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/kpartx/Makefile b/kpartx/Makefile
index bfa6fe8..be87124 100644
--- a/kpartx/Makefile
+++ b/kpartx/Makefile
@@ -8,9 +8,7 @@ LDFLAGS += $(BIN_LDFLAGS)
 
 LIBDEPS += -ldevmapper
 
-ifneq ($(call check_func,dm_task_set_cookie,/usr/include/libdevmapper.h),0)
-	CFLAGS += -DLIBDM_API_COOKIE
-endif
+CFLAGS += -DLIBDM_API_COOKIE
 
 OBJS = bsd.o dos.o kpartx.o solaris.o unixware.o dasd.o sun.o \
 	gpt.o mac.o ps3.o crc32.o lopart.o xstrncpy.o devmapper.o
-- 
2.8.1

